universal7885-common/BoardConfigCommon.mk

```diff
@@ -96,6 +96,11 @@ TARGET_VENDOR_PROP += $(COMMON_PATH)/configs/vendor_64.prop
 else
 TARGET_VENDOR_PROP += $(COMMON_PATH)/configs/vendor_32.prop
 endif
+ifeq ($(TARGET_TEE_KERNEL),teegris)
+TARGET_VENDOR_PROP += $(COMMON_PATH)/configs/efspie.prop
```

Comment:
init.baseband.rc needs commonization.
Should be easy to write small program that does its job, it is just few mounts and permission changes.